### DISCUSSION TITLE
Can you use cctags on your rig?

### DISCUSSION THREAD

**Discussion by hall488** [alicevision/Meshroom#2475] [Q&A]
Hello, I am using a robot with multiple cameras on it to reconstruct a pipe it is traveling through. Is it possible to place cctags on my robot and still properly scale the model? The cctags would be in the same position on each frame for each camera.

**Comment by natowi** [comment]
You want to avoid parts in an image that stay static. You could however generate a mask and mask out the static area.

**Reply by hall488** [reply]
I think my question was poorly phrased, would cctags still work if they are static across all frames

**Reply by natowi** [reply]
It can be used but depending on what you are trying to achieve you have to configure your nodes accordingly.
I would set up a graph with masking to mask out the static area. Then use one set of photos from the camera rig in a separate graph without masking and cctag detection + rotation/scaling if desired. Both graphs are then merged by sfm alignment, using the cctag reonstruction as target and the full masked reconstruction as source.
